src/lambda/explorer/cloudFormationNodes.ts

```diff
@@ -18,7 +18,9 @@ import { ErrorNode } from '../../shared/treeview/nodes/errorNode'
 import { PlaceholderNode } from '../../shared/treeview/nodes/placeholderNode'
 import { intersection, toArrayAsync, toMap, toMapAsync, updateInPlace } from '../../shared/utilities/collectionUtils'
 import { listCloudFormationStacks, listLambdaFunctions } from '../utils'
-import { FunctionNodeBase } from './functionNode'
+import { LambdaFunctionNode } from './lambdaFunctionNode'
+
+export const CONTEXT_VALUE_CLOUDFORMATION_LAMBDA_FUNCTION = 'awsCloudFormationFunctionNode'
```

Comment:
Would it be better to add this (and `CONTEXT_VALUE_LAMBDA_FUNCTION`) as a public readonly in the classes themselves? I feel like this is only exported for testing purposes.

Reply:
I wouldn't move them into `LambdaFunctionNode` because what `contextValue` is set to is not a concern of that class. If you were asking about moving the constant into (for example) the CloudFormation stack node, I can't see the advantage of either way, since it is located/exported from the same file.

Reply:
Ah, duh, forgot that it was creating the LambdaFunctionNode (thought CloudFormationNode was an impl of LambdaFunctionNode). It would've been one less import statement for consumers down the line, but that's fine. Disregard.